docs/dev/Lua API.rst

```diff
@@ -6622,6 +6622,14 @@ tiletypes
   the parameters are enum values from ``df.tiletype_shape``,
   ``df.tiletype_material``, etc. Returns whether the conversion succeeded.
 
+* ``bool tiletypes_setTile(pos, tiletype_options)`` where
+  the ``tiletype_options`` parameter takes in a table, with any fields
+  matching the available tiletypes options:
+  ``shape``, ``material``, ``special``, ``variant``, ``dig``,
```

Comment:
what does `dig` mean? I don't see it actually being used for anything. If it's intended to set a dig designation, I'd advocate against adding that feature to `tiletypes`. I think it's out of scope. `tiletypes` *does* have a *filter* for dig designated tiles, but that doesn't seem to be relevant in a function called `setTile` (unless it supports filters)

Reply:
I'd just included it since I saw it in the TileTypes struct definition, probably should've thought it through a bit more though now that you mention it, cause yeah, it doesn't make any actual sense to include it.

Reply:
I just checked again to make sure, but the `dig` option is available from the command line `tiletypes` function in its current state, should it be removed from there as well?
![image](https://github.com/DFHack/dfhack/assets/26845479/be13a6ba-96e2-44de-8e62-557ef1b26a58)

Reply:
Yeah, the code doesn't actually do anything if you paint `designated 1`. I'd remove it.